mlflow/pytorch/__init__.py

```diff
@@ -46,8 +46,30 @@
 
 def get_default_conda_env():
     """
-    :return: The default Conda environment for MLflow Models produced by calls to
+    :return: The default Conda environment as a dictionary for MLflow Models produced by calls to
              :func:`save_model()` and :func:`log_model()`.
+
+    .. code-block:: python
+        :caption: Example
+
+        import mlflow.pytorch
+
+        # Log PyTorch model
+        with mlflow.start_run() as run:
+            mlflow.pytorch.log_model(model, "models_pth")
```

Comment:
Should we remove `_pth` here?

Reply:
Removed other instances and usage of `_pth` suffix 👍